CONTRIBUTING.md

```diff
@@ -250,20 +250,12 @@ confused with 3-letter acronyms.
 
 ## Design Proposals
 
-Solana's architecture is described by docs generated from markdown files in
```

Comment:
@mvines wdyt about this change? Gave a more thorough explanation in the core docs on implementing proposals and linked in `CONTRIBUTING.md`.

Reply:
sure